xmodule/static_content.py

```diff
@@ -132,7 +132,9 @@ def _write_styles(selector, output_root, classes, css_attribute, suffix):
     for class_ in classes:
         class_css = getattr(class_, css_attribute)()
         for filetype in ('sass', 'scss', 'css'):
-            for idx, fragment in enumerate(class_css.get(filetype, [])):
+            for idx, fragment_path in enumerate(class_css.get(filetype, [])):
+                with open(fragment_path, 'rb') as fragment_file:
+                    fragment = fragment_file.read()
```

Comment:
When processing the SCSS, we now must load the files rather than just looking at the contents.

This creates a very minor build-time inefficiency: any SCSS file that is used across multiple XModules will be loaded multiple times. On the other hand, it achieves a minor run-time optimization: these SCSS files are no longer loaded along with XModule classes.

The inefficiency will be gone soon, because follow-up refactorings aim to delete this processing step.